components/telemetry-controller/PROJECT

```diff
@@ -0,0 +1,15 @@
+domain: kyma-project.io
+layout:
+- go.kubebuilder.io/v3
+projectName: telemetry-controller
+repo: github.com/kyma-project/kyma/components/telemetry-controller
```

Comment:
As discussed, maybe let's rename it to `telemetry-operator` to have some ubiquitous language here

Reply:
Done